q2-structure/proposal-by-xdg.md

```diff
@@ -0,0 +1,222 @@
+id="note1ref"><sup>1</sup></a> The council members will serve until the
+completion of the next Term Election unless they resign or are removed.
+
+Whenever there are vacancies on the Steering Council, the council will
+organize a Special Election within one week after the vacancy occurs.  If the
+entire Steering Council is ever vacant, a Term Election will be held instead.
+
+The election of the first Steering Council will be a Term Election.  Ricardo
+Signes will be the Vote Administrator for the initial Term Election unless he
+is a candidate, in which case he will select a non-candidate administrator to
+replace him.
+
+### Resignation, Absence, and Removal
+
+Steering Council members may resign their position at any time.
+
+If a Steering Council member is deceased, or drops out of touch and cannot be
+contacted for a month or longer, then the rest of the council may vote to
+declare their seat vacant.  If an absent member returns after such a
+declaration is made, they are not reinstated automatically, but may run in the
+Special Election to fill the vacancy.
+
+Otherwise, Steering Council members may only be removed before the end of
+their term by a No Confidence Vote by the Core Team.
+
+## Voting
+
+### Vote Administration
+
+Every vote requires a Vote Administrator who manages communication, collection
+of secret ballots, and all other necessary activities to complete the voting
+process.
+
+Unless otherwise specified, the Steering Council selects the Vote
+Administrator.
+
+A Vote Administrator must not be a member of the Steering Council nor a
+candidate or subject of the vote.  A Vote Administrator may be a member of the
+Core Team and, if so, may cast a vote while also serving as administrator.  If
+the Vote Administrator becomes a candidate during an election vote, they will
+appoint a non-candidate replacement.
+
+If the entire Steering Council is vacant or is the subject of a No Confidence
+Vote, then the Core Team will select a Vote Administrator by consensus.  If
+consensus cannot be reached within one week, the President of The Perl
+Foundation will select a Vote Administrator.
+
+### Term and Special Elections
+
+Term and Special Election are ranked-choice votes to construct an ordered list
+of candidates to fill vacancies in the Steering Council.
+
+Both Term and Special Elections occur in two stages:
+
+1. Candidates advertise their interest in serving. Candidates must be
+   nominated by a Core Team member. Self-nominations are allowed.  Nominated
+   candidates may share a statement about their candidacy with the Core Team.
+
+2. Active Core Team Members vote by ranking all candidates.  Voting is
+   performed anonymously.  After voting is complete, candidates are ranked
+   using the Condorcet Minimax method.  If a tie occurs, it may be resolved by
+   mutual agreement among the tied candidates, or else the tie will be
+   resolved through random selection by the Vote Administrator.
```

Comment:
I get the desire to simplify but presumably this is going to be rare. It would only be relevant if 5 and 6 candidates get the same vote count. IMO a lot of questions will be asked about "random" if this happens.

Reply:
This is straight from PEP 13, except I've specified the Vote Administrator is the one who randomizes.  I'm going to leave this as is.

Reply:
Fair enough. An alternate suggestion might be that the vote admin gets to choose the winner of a tie and not specify how?